client/main.js

```diff
@@ -41,21 +42,29 @@ Template.startTimer.helpers({
 });
 
 Template.startTimer.events({
-    'click .js-play'(event, instance) {
+    'click .js-play' (event, instance) {
         instance.playerHandler.set(Meteor.setInterval(() => {
             instance.timer.set(instance.timer.get() + 1000);
             console.log(instance.timer.get());
         }, 1000));
 
     },
-    'click .js-pause'(event, instance) {
+    'click .js-pause' (event, instance) {
         Meteor.clearInterval(instance.playerHandler.get());
         instance.playerHandler.set(null);
     },
-    'click .js-back'(event, instance) {
+    'click .js-back' (event, instance) {
         instance.timer.set(instance.timer.get() - 1000);
     },
-    'click .js-forward'(event, instance) {
+    'click .js-forward' (event, instance) {
         instance.timer.set(instance.timer.get() + 1000);
     },
-});
+    'click .lampada-js' (event) {
+        element = event.currentTarget;
+        if (element.src.match("lampada-on")) {
+            element.src = "https://www.devfuria.com.br/javascript/code-lampada/lampada-off.gif";
```

Comment:
you should use if/else on html instead of changing the img like this, also you should put the image inside our assets in public folder instead of using directly from a website